src/lib/operators/aggregate/window_function_traits.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "expression/window_function_expression.hpp"
```

Comment:
No, this header is actually needed, because types like `WindowFunction` are used. In fact, it only worked by coincidence before (because everyone using these traits was also including `WindowFunction` beforehand)